ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/app/server.clj

```diff
@@ -79,7 +80,8 @@
      (-> (create-ring-handler)
          (httpkit/start-container port))
      (-> (create-ring-handler)
-         (aleph/start-container port)))))
+         (aleph/start-container port)))
+   (graphite/start-graphite-reporter)))
```

Comment:
The start of graphite reporter should probably be done before the start of the web container.
This modifies the return value of this function (no longer the expected and documented stop function).